### DISCUSSION TITLE
Grib2 data format in mikeio

### DISCUSSION THREAD

**Discussion by spurgeondsk** [DHI/mikeio#257] [Q&A] [answered]
Is there any way to work with grib2 data format?
I tried to do xarray with cfgrib but couldn't succeed in windows due to some dependency issues. Looking for some alternative way. Any suggestions

**Comment by ecomodeller** [comment, marked as answer]
Well, grib2 is out of scope for MIKE IO.

Xarray and cfgrib should be the way to go, but I guess most grib users are using Linux.

An alternative approach is to use wgrib2 and convert grib2 to netcdf or to extract data in binary format
https://www.cpc.ncep.noaa.gov/products/wesley/wgrib2/netcdf.html

**Reply by spurgeondsk** [reply]
Thank you, converting to NetCDF using wgrib2 and then to dfs2 is pretty easy and workable in the windows platform where MIKEIO tools work.

**Reply by spurgeondsk** [reply]
Any code already developed for converting ROMS model output to dfs2/dfs3. I just working on it.

**Comment by jsmariegaard** [comment]
You could also consider [WSL](https://docs.microsoft.com/en-us/windows/wsl/) for your linux needs

**Reply by spurgeondsk** [reply]
Yes that too a good option.
